scripts/javascript-workings.js

```diff
@@ -0,0 +1,75 @@
+const parentNode = document.querySelector(".news--area--feed");
+
+// compile this Url = "https://newsapi.org/v2/top-headlines?sources=bbc-news,daily-mail,mirror,&apiKey=93238bcda39e4404852697d364b77971";
+const baseUrl = "https://newsapi.org/v2/top-headlines?apiKey=93238bcda39e4404852697d364b77971";
+
+
+const publicationList = {
+    "bbc-news": true,
+    "daily-mail": true,
+    "mirror": true
+}
+
+// create an array from object using key values
+let publicationArray = Object.keys(publicationList);
+
+
+//console.log(defaultArray);
+const sourceList = `&sources=${defaultArray}`;
+const fullURL = `${baseUrl}${sourceList}`;
+
+function displayDataOnPage(newsStories) {
+  const newsArray = newsStories.articles;
+  console.log(newsStories.articles);
```

Comment:
There are quite a few consol.logs left in. Once you are happy with the functionality, it's best to remove them to keep console output clean